test/mp/test/test.cpp

```diff
@@ -60,6 +60,7 @@ class TestSetup
 
     TestSetup(bool client_owns_connection = true)
         : thread{[&] {
+
```

Comment:
In commit "eventloop: add `LogOptions` struct" (d9eb7fb8cbeb58d3843f3d32974831b26613bc92)

Note: extra newline is added here, should be no need to change this file anymore